VERSION.cmake

```diff
@@ -1,6 +1,6 @@
 SET( VERSION_MAJOR "3")
-SET( VERSION_MINOR "4" )
-SET( VERSION_PATCH "2" )
+SET( VERSION_MINOR "5" )
+SET( VERSION_PATCH "0" )
 SET( VERSION "${VERSION_MAJOR}.${VERSION_MINOR}.${VERSION_PATCH}${GIT_SHA1_VERSION}" )
 
 ##### This is need for the libyui core, ONLY.
```

Comment:
This changes the ABI and makes the package incompatible with the previous versions we should bump the .so version and the package name (e.g. `libyui10`). As described below we need to do it in all packages.

SET( SONAME_MAJOR "10" )